api/src/index.js

```diff
@@ -5,12 +5,77 @@ import bodyParser from "body-parser";
 import knex from "./database_client.js";
 import nestedRouter from "./routers/nested.js";
 
+const meals = await knex.raw("SELECT * FROM Meal");
+console.log(meals);
+
 const app = express();
 app.use(cors());
 app.use(bodyParser.json());
 
 const apiRouter = express.Router();
 
+app.get("/my-route", (req, res) => {
+  res.send("Hi friend");
+});
+
+// Respond with all meals in the future (relative to the 'when' datetime)
+app.get('/future-meals', async (req, res) => {
+  try {
+    const meals = await knex.raw("SELECT * FROM Meal WHERE `when` > NOW()");
+    res.json(meals[0].length ? meals[0] : []); // meals[0] because knex.raw returns an array
+  } catch (err) {
+    res.status(500).json({ error: 'Internal server error' });
+  }
+});
+
+// Respond with all meals in the past (relative to the 'when' datetime)
+app.get('/past-meals', async (req, res) => {
+  try {
+    const meals = await knex.raw("SELECT * FROM Meal WHERE `when` < NOW()");
+    res.json(meals[0].length ? meals[0] : []); // Return empty array if no meals
+  } catch (err) {
+    res.status(500).json({ error: 'Internal server error' });
+  }
+});
+
+// Respond with all meals sorted by ID
+app.get('/all-meals', async (req, res) => {
+  try {
+    const meals = await knex.raw("SELECT * FROM Meal ORDER BY id");
```

Comment:
Using [Knex Query Builder](https://knexjs.org/guide/query-builder.html) we could simplify this to:
```suggestion
    const meals = await knex("Meal").orderBy("id");
```